ern-container-gen/src/generators/ios/hull/ElectrodeContainer.xcodeproj/project.pbxproj

```diff
@@ -416,16 +420,14 @@
 				STRIP_STYLE = debugging;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.0;
 			};
 			name = QADeployment;
 		};
 		303268E61EA8139700A41A8F /* QADeployment */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)"
-				);
+				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
```

Comment:
Please update all `FRAMEWORK_SEARCH_PATHS` in this file as they were before this commit (i.e surrounded by parenthesis). I think that our pbxproj parser is quite picky on this :(
Don't have time atm to look in the parser and ways to improve it, so just to be on the safe side let's just format these the way they were.